canonical-web-and-design/microstack.run
# Copy update - "MicroStack - OpenStack in a snap"

## Done

- Update index.html page according to [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8039/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check instructions in "Get started with MicroStack" for linux, windows and mac match [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit)


## Issue / Card

Fixes [#8348](https://github.com/canonical-web-and-design/ubuntu.com/issues/8348)
